### PR TITLE
Feeling room initial angry animation

### LINKED ISSUE #43: Feel Room : Anger : Base animation
Basic idea and implementation of angry animation view in the FeelingRoom

### PR DESCRIPTION
**Pull request by Kristiannmiller**
## Description
This PR:
- Adds base animation of dropping/exploding squares to the angry FeelingRoom view
- Will need additional styling (including background gradient animation)
- Will need some additional media queries to handle mobile views
- Will need some additional styling to handle smaller squares rendering in unexpected locations further into the animation process (typically when window is in a mobile view)

## Motivation and Context
- closes #43 

## How Has This Been Tested?
- [X] In console
- [X] In browser

## Types of changes

- [X] New feature (non-breaking change which adds functionality)
- [X] Styling / Design

## Checklist:

- [X] My code follows the code style of this project.
- [X] My change requires a change to the documentation (README)

### CODE REVIEW

File: src/Components/FeelingRoom/FeelingRoom.js [hunk 1]
```diff
@@ -31,18 +33,30 @@ class FeelingRoom extends Component {
     )
   }
   createAngryRoom = () => {
-    //functions to animate go here
-    return (
-      console.log('ANGRY')
-      //elements to animate go here
+    const squareNums = Array.from(Array(10).keys())
+      angryFeelingAnimation()
+    const angrySquares = squareNums.map(num => {
```
Comment (BrigetteDoelp) [inline src/Components/FeelingRoom/FeelingRoom.js:66]: I can not begin to articulate how much I love the angry squares. Your animation is GREAT, and really reflective of the vibe we hoped would be created in this room.

File: src/Components/FeelingRoom/angryAnimations.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
+import gsap from 'gsap'
+
+export const angryFeelingAnimation = () => {
+  let angryTimeline = gsap.timeline({repeat: 10});
+  for (var i = 0; i <= 9; i++) {
+    angryTimeline.from(`.square${i}`, {
```
Comment (BrigetteDoelp) [inline src/Components/FeelingRoom/angryAnimations.js:6]: Really impressive use of the `timeline` functionality of `gsap`!

File: src/Components/FeelingRoom/angryAnimations.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    .to(`.square${i} #part2`, {
+      backgroundColor: 'hsla(329, 57%, 25%)',
+    }, "shake")
+    .to(`.square${i} #part3`, {
+      backgroundColor: 'hsla(329, 57%, 25%)',
+    }, "shake")
+    .to(`.square${i} #part4`, {
+      backgroundColor: 'hsla(329, 57%, 25%)',
+    }, "shake")
+    .add("explode")
```
Comment (BrigetteDoelp) [inline src/Components/FeelingRoom/angryAnimations.js:33]: Ah, I see, this is how you layer on the animations and such in the timeline! This looks really, really good.

File: src/Components/FeelingRoom/FeelingRoom.js (hunk 1 above)
Comment (nathanielmillard) [reply src/Components/FeelingRoom/FeelingRoom.js:66]: I agree! The emotion of this room is great.

File: src/Components/FeelingRoom/angryAnimations.js [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+      rotate: "random(-45, 45, 2)",
+      backgroundColor: 'hsla(225, 10%, 8%, 0%)'
+    }, "explode")
+    .to(`.square${i} #part3`, {
+      y: "-=50vh",
+      x: "-=60vh",
+      duration: .5,
+      rotate: "random(-45, 45, 2)",
+      backgroundColor: 'hsla(225, 10%, 8%, 0%)'
+    }, "explode")
```
Comment (nathanielmillard) [inline src/Components/FeelingRoom/angryAnimations.js:58]: I do wonder if there are ways shorten this code, and have less lines.
But working correctly and written out extensively is the best place to start for sure.

**Review by nathanielmillard — APPROVED** [review]
I know you are still going to pour a lot of work into this, but I have to say that the animation looks fantastic already.

**Pull request merged**